tsconfig.json

```diff
@@ -12,9 +12,9 @@
     "forceConsistentCasingInFileNames": true,
     "noEmit": true,
     "esModuleInterop": true,
-    "module": "esnext",
+    "module": "NodeNext",
     // Need this line for the import @storyblok to work
-    "moduleResolution": "nodenext",
+    "moduleResolution": "NodeNext",
```

Comment:
Eslint complains that if `moduleResolution` is set to `NodeNext`, `module` needs to be the same